### ISSUE TITLE
Port to 1.21 / Neoforge?

### ISSUE THREAD

**Issue by xjqsh** [MCModderAnchor/TACZ#225]
Since key features of tacz are not yet developed, there are **NO PLANS** to port to a higher version in the near future.  
Please **don't open any issues about version porting**, it's very disturbing to us, and they will be automatically ignored and closed.

**UPD 2025/09/09**
We plan to officially port TACZ to a higher version after completing the next major update (likely the final major one), and future updates will primarily focus on maintenance.

Howerver, since almost all the core members of the development team have graduated and are busy with their works, the current development progress is extremely slow. So before this major update arrives, there may be several smaller updates, but these will only include new weapons or items and some fixes, but no new features.

Before that, if you really want to play the 1.21 version of TACZ, there's an unofficial port of TACZ made by @MUKSC, you can get it from [this repo](https://github.com/MUKSC/TACZ-1.21.1) / [curseforge](https://www.curseforge.com/minecraft/mc-mods/tacz-1-21-1) or join our discord and find the post, but we cannot guarantee future compatibility (including saves and gun packs, etc.).

**Comment by IGN-Styly** [comment]
Is there any road map to follow progress? 1.21 will give full DTH and Sodium compat.

**Comment by johnfromstatefarm** [comment]
Pls update, Vic's Point Blank can't be the only gun mod in 1.21. I hate the crafting system 😕

**Comment by blackmidiuser** [comment]
please. i dont use vics point blank.

**Comment by blackmidiuser** [comment]
> please. i dont use vics point blank.

Vic's point blank caused the water to be red. just PLEASE. WE NEED THIS IN 1.21

**Comment by shotadft** [comment]
From me as well, I love the animation of TaCZ!

**Comment by yogiyogi0530-byte** [comment]
Yogi

**Comment by IGN-Styly** [comment]
Can someone lock the conversation and give a statement?

**Comment by CitiesXL2815** [comment]
> Can someone lock the conversation and give a statement?

I agree, Locking this would be Great, it's all Beggar's wanting them to port, @xjqsh i would Lock this, you made Statement Above, that should be it, Locking this is the best Option, People need to Know, When they eventually have Plans to Port to Higher Versions, they Might give Announcement on Discord or other platform they use and etc. they will port it when they Want too, up to the devs not you
